spec/models/case/sar/offender_spec.rb

```diff
@@ -777,5 +777,43 @@
       expect(closed_kase.num_days_late).to eq 2
     end
   end
-  
+
+  describe '#first_prison_number' do
+    let(:kase)  { create :offender_sar_case }
+
+    context 'When there is just one prison number' do
+      it 'returns the prison number' do
+        expect(kase.first_prison_number).to eq '123465'
+      end
+    end
+
+    context 'When there is more than one prison number' do
+      let(:kase)  { create :offender_sar_case, prison_number: 'A12345, B98765' }
+      let(:kase1)  { create :offender_sar_case, prison_number: '  A12345  , B98765 ' }
+      let(:kase2)  { create :offender_sar_case, prison_number: 'A12345 B98765' }
```

Comment:
What about `A12345, B98765 A234667`, is this a possible string? (i.e. mix of commas and spaces)